runner-plugins/runner-scribe/main.go

```diff
@@ -85,7 +86,10 @@ func main() {
 }
 
 func sendVulnerability(item gozdef.VulnEvent) (err error) {
-	ac := gozdef.APIConf{URL: conf.MozDef.URL}
+	ac := gozdef.APIConf{
```

Comment:
This should be `APIConf {` per surrounding examples (such as line 110).

Reply:
This is the format enforced by `gofmt`.